src/patternfly/components/Icon/icon.scss

```diff
@@ -42,6 +42,12 @@
   width: var(--#{$icon}--Width);
   height: var(--#{$icon}--Height);
 
+  @include pf-v5-mirror-inline-on-rtl;
+
+  &.pf-m-dir-ltr {
+    direction: ltr;
+  }
```

Comment:
I don't think we want to reverse anything by default, and we have the global modifier that can be applied to reverse an icon (or anything else).